plugin-server/src/main/ingestion-queues/session-recording/services/session-manager.ts

```
@@ -417,6 +418,7 @@ export class SessionManager {
             })
             this.captureException(error)
             counterS3WriteErrored.inc()
+            this.s3WriteFailed = true
```

let's mark the session manager when the error occurs

---

hmmm, if we were already anyway swallowing errors do we even need this